<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistSearchDialog</class>
 <widget class="QDialog" name="PlaylistSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="caseCheckBox">
        <property name="text">
         <string>Case sensitivity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchLineEdit"/>
   </item>
   <item>
    <widget class="TableViewEx" name="searchTableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableViewEx</class>
   <extends>QTableView</extends>
   <header location="global">src/gui/widget/tableviewex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
